include/graph.h

```diff
@@ -3,15 +3,20 @@
 #define INCLUDE_GRAPH_H_
 
 #include <unordered_map>
-#include <vector>
 #include <utility>
+#include <vector>
 
 class Graph {
-    public:
-        int size;
-        std::unordered_map<int, std::vector<std::pair<int, int>>> adjList;
+private:
+  int size;
```

Comment:
nit: use some naming convention for private members like underscore prefix (e.g. `_size`) so your getters don't have to be things like `getSize()` but rather `size()`